src/api/admin.ts

```diff
@@ -513,7 +520,8 @@ const createFirestoreRoomInput_v2 = async (
     url:
       input.useUrl || !input.venueName
         ? input.url
-        : window.origin + venueInsideUrl(input.venueName),
+        : window.origin +
+          attendeeSpaceInsideUrl(world.slug, input.venueName as SpaceSlug),
```

Comment:
Where possible we should be casting to the appropriate slug type as soon as we get it: e.g. from user input, URL or database. Then it just becomes "the right thing" thereafter. I think we probably need to convert more of this type of code to using a `generateSlug` function instead of just passing name through blindly. I would do that here, but, I'm wary of increasing the scope of this PR even further.